requirements.txt

```diff
@@ -1 +1,7 @@
+flake8
```

Comment:
Shouldn't these be in a `requirements-dev.txt`?

Reply:
I'll add the dev-requirements.txt file